WordPress/Classes/Models/Blog+Quota.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+extension Blog {
+
+    @objc var isQuotaAvailable: Bool {
+        return quotaSpaceAllowed != nil && quotaSpaceUsed != nil
+    }
+
+    @objc var quotaPercentangeUsed: NSNumber? {
+        guard let quotaSpaceAllowed = quotaSpaceAllowed?.int64Value, let quotaSpaceUsed = quotaSpaceUsed?.int64Value else {
```

Comment:
We could ask for the `doubleValue` for these numbers directly here, instead of having to cast them later. We should probably also check that `quotaSpaceAllowed` is `> 0`, so we don't try and divide by zero.

Reply:
I wanted to use Double? but then the compiler complains that this will not work in ObjC.

Reply:
You can keep the method returning `NSNumber`, but just switch out the `guard let` to get the double value instead of int64 – something like this:

```
@objc var quotaPercentageUsed: NSNumber? {
    guard let quotaSpaceAllowed = quotaSpaceAllowed?.doubleValue, let quotaSpaceUsed = quotaSpaceUsed?.doubleValue, quotaSpaceAllowed > 0 else {
        return nil
    }

    return NSNumber(value: quotaSpaceUsed / quotaSpaceAllowed)
}
```

Reply:
Ah ok I thought you where referring to the method signature.